### PR TITLE
Workshop Addition

### PR DESCRIPTION
**Pull request by chrisking**
*Issue #, if available:*

*Description of changes:*
Restructured the workshop content to fit in with the existing repository. Also built as a feature branch inside this repo not a fork.

Upon acceptance the old repo within my account will be updated to just contain a pointer to this repository.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: getting_started/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+[![Launch Stack](https://s3.amazonaws.com/cloudformation-examples/cloudformation-launch-stack.png)](https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=PersonalizeDemo&templateURL=https://chriskingpartnershare.s3.amazonaws.com/PersonalizeDemo.yaml)
+
+Follow along with the screenshots if you have any questions about these steps.
+
+### Cloud Formation Wizard
+
+Start by clicking `Next` at the bottom like shown:
+
+![StackWizard](static/imgs/img1.png)
```
Comment (yifeim) [inline getting_started/README.md]: Reduce image size.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 Notebooks and examples on how to onboard and use various features of Amazon Personalize
 
-## License Summary
+## Getting Started Workshop
+
+Open the `getting_started` folder to find a CloudFormation template that will deploy all the resources you need to build your first campaign with Amazon Personalize. The notebooks provided can also serve as a template to building your own models with your own data. 
+
+This repository is cloned into the environment so you can explore the more advanced notebooks with this approach as well.
```
Comment (yifeim) [inline README.md]: Also mention that a minimal script is available in [personalize_sample_notebook.ipynb](https://github.com/aws-samples/amazon-personalize-samples/blob/master/personalize_sample_notebook.ipynb).

File: getting_started/README.md (hunk 1 above)
Comment (chrisking) [reply getting_started/README.md]: Can you elaborate on the requirement here? They are pretty standard sizes for following along on the page in GitHub.

They've worked well for a few hundred users at this point.

File: README.md (hunk 2 above)
Comment (chrisking) [reply README.md]: Completed.

File: getting_started/README.md (hunk 1 above)
Comment (yifeim) [reply getting_started/README.md]: This is what I am thinking: reduce to width="75%" so the flow can be clearly presented.

Comment (yifeim) [reply getting_started/README.md]: https://gist.github.com/uupaa/f77d2bcf4dc7a294d109

Comment (chrisking) [reply getting_started/README.md]: Right, that is not actively supported by GH markdown for percentages. Exhibited by the fact it isn't rendered on the test page. I think we can do without that optimization at this time.

**Pull request merged**